tidy3d/plugins/design/parameter.py

```diff
@@ -51,22 +51,26 @@ def _sample_grid(self) -> List[Any]:
     def select_from_01(self, pts_01: np.ndarray) -> List[Any]:
         """Select values given a set of points between 0, 1."""
 
+    @abstractmethod
+    def sample_first(self) -> Any:
+        """Output the first allowed sample."""
+
 
 class ParameterNumeric(Parameter, ABC):
     """A variable with numeric values."""
 
     span: Tuple[Union[float, int], Union[float, int]] = pd.Field(
         ...,
         title="Span",
-        description="(min, max) inclusive range within which the variable should be swept.",
+        description="(min, max) inclusive range within which are allowed values for the variable.",
```

Comment:
should probably mention whether these bounds are inclusive? seems I forgot in the original description

Reply:
same comments below